src/collectors/mongodb/mongodb.py

```diff
@@ -177,6 +177,10 @@ def collect(self):
             self._publish_transformed(data, base_prefix)
             if str_to_bool(self.config['simple']):
                 data = self._extract_simple_data(data)
+            if str_to_bool(self.config['replica']):
+                replset_data = conn.admin.command('replSetGetStatus')
```

Comment:
how to gracefully handle if replset isn't yet initiated? might give a warning msg